README.md

```diff
@@ -366,11 +380,13 @@ Authors & Contributors
 
 This software was developed at [Open Systems AG](https://www.open.ch/) in close collaboration with the [Distributed Computing Group](http://www.disco.ethz.ch/) at the [Swiss Federal Institute of Technology](https://www.ethz.ch/en.html).
 
+This repository has been forked off the Open Systems repository end of 2018. Bug fixing and development of new features is done in this repository. The Open Systems one can be considered unmaintained.
```

Comment:
Maybe we should consider requesting to mark this repository as standalone instead of a fork (https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile -> Detach a fork), since we're obviously maintaining the status quo which can't be merged back without significant effort on the OSAG side... Just an idea...

Reply:
I like the idea. Dunno if you can see it: https://support.github.com/ticket/personal/0/833989. Will keep you posted.

Reply:
Nice one, saw the confirmation, love it! Now officially standalone :muscle:

Reply:
Done